orttraining/orttraining/core/graph/optimizer_graph_builder.cc

```diff
@@ -377,10 +373,12 @@ OptimizerGraphBuilder::OptimizerGraphBuilder(
     const OptimizerBuilderRegistry& opt_builder_registry,
     const OptimizerGraphConfig& opt_graph_config,
     const std::unordered_map<std::string, OptimizerNodeConfig>& weight_names_to_opt_configs,
-    std::unordered_map<std::string, std::string>& updated_weight_names_map)
+    std::unordered_map<std::string, std::string>& updated_weight_names_map,
+    std::unordered_map<std::string, TrainingSession::PartitionInfo>& weight_partition_info)
```

Comment:
can we add the field for updated name within weight_partition_info to avoid passing two parameters that will have associated information

Reply:
Sync offline, will add in next PR since this is a backend refactor and does not affect front end